src/demo-app/app/github-issues/splitter/split-handle.directive.ts

```diff
@@ -1,36 +1,30 @@
-import { Directive, ElementRef, Output } from '@angular/core';
-import { Observable } from 'rxjs/Observable';
-import 'rxjs/add/operator/takeUntil';
+import {Directive, ElementRef, Output} from '@angular/core';
+import {Observable} from 'rxjs/Observable';
+
 import 'rxjs/add/observable/fromEvent';
 import 'rxjs/add/operator/takeUntil';
 import 'rxjs/add/operator/switchMap';
+import 'rxjs/add/operator/map';
 
 @Directive({
   selector: '[ngxSplitHandle]',
   host: {
     class: 'ngx-split-handle',
-    title : 'Drag to resize'
+    title: 'Drag to resize'
   }
 })
 export class SplitHandleDirective {
 
   @Output() drag: Observable<{ x: number, y: number }>;
 
   constructor(ref: ElementRef) {
-    const getMouseEventPosition = (event: MouseEvent) => ({ x: event.movementX, y: event.movementY });
+    const getMouseEventPosition = (event: MouseEvent) => ({x: event.movementX, y: event.movementY});
 
-    const mousedown$ = Observable.fromEvent(ref.nativeElement, 'mousedown').map(getMouseEventPosition);
+    const mousedown$ = Observable.fromEvent(ref.nativeElement, 'mousedown').map(getMouseEventPosition); // tslint:disable-line:max-line-length
     const mousemove$ = Observable.fromEvent(document, 'mousemove').map(getMouseEventPosition);
     const mouseup$ = Observable.fromEvent(document, 'mouseup');
 
-    this.drag = mousedown$
-        .switchMap(mousedown =>
-          mousemove$.map(mousemove => ({
-            x: mousemove.x,
-            y: mousemove.y
-          }))
-          .takeUntil(mouseup$)
-        );
+    this.drag = mousedown$.switchMap(_ => mousemove$.takeUntil(mouseup$));
```

Comment:
We don't use the trailing `$` in Angular or in Material. This would be good to change in a follow-up PR.